apps/ui/src/networks/types.ts

```diff
@@ -181,9 +181,9 @@ export type NetworkApi = {
     filter?: 'any' | 'for' | 'against' | 'abstain',
     sortBy?: 'vp-desc' | 'vp-asc' | 'created-desc' | 'created-asc'
   ): Promise<Vote[]>;
-  loadUserVotes(spaceId: string, voter: string): Promise<{ [key: string]: Vote }>;
+  loadUserVotes(spaceId: string | undefined, voter: string): Promise<{ [key: string]: Vote }>;
```

Comment:
This is not optimal, as it can potentially load huge list for heavy voters, but it's rare.

Reply:
We could refactor the function to take a `filter` argument, like `loadSpaces`, and have something like this:

```
export type UserVotesFilter = {
  spaceId?: string[];
};

loadUserVotes: async (
  voter: string,
  filters?: UserVoteFilters
): Promise<{ [key: string]: Vote }>
```

Reply:
Removed the need, by always requiring presence of space ids.

Instead of loading the user's votes without `spaceId` for the timeline page, we're now loading only votes for spaces in their follow list.

Instead of `string | undefined`, arg type is now `string[]`